vm/src/stdlib/os.rs

```diff
@@ -113,6 +113,25 @@ fn os_error(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
     Err(vm.new_os_error(msg))
 }
 
+fn os_read(vm: &VirtualMachine, args: PyFuncArgs) -> PyResult {
```

Comment:
These functions probably would be easy to write with new args style, something like:

```
fn os_read(fd: PyIntRef, n: PyIntRef, vm: &VirtualMachine) -> PyResult
```

Reply:
Done.